image/rhel/konflux.Dockerfile

```diff
@@ -178,4 +140,4 @@ RUN chown -R 4000:4000 /etc/pki /etc/ssl && save-dir-contents /etc/pki/ca-trust
     mkdir -p /var/cache/stackrox && chown -R 4000:4000 /var/cache/stackrox && \
     chown -R 4000:4000 /tmp
 
-USER 4000:4000
+USER 4000:4000
```

Comment:
Somehow your editor removed a trailing newline so that GitHub shows a red sign here.